tuxemon/states/phone/phone_banking.py

```diff
@@ -33,99 +33,135 @@ def add_menu_items(
     ) -> None:
         if "bank_account" not in self.player.money:
             self.player.money["bank_account"] = 0
-        bank = self.player.money["bank_account"]
-        money = self.player.money["player"]
+        if "bill_cathedral" not in self.player.money:
```

Comment:
this kind of mod-specific code should be avoided.  i think its ok for now since it is labeled "cathedral", but if we expand on the banking system in the future, we should plan out a way to do it with plugins

Reply:
I'm open to plugin, but I don't know if you have some "rule" or method to do it.